tests/test_sort.rs

```diff
@@ -71,6 +71,41 @@ fn test_multiple_files() {
 }
 
 #[test]
+fn test_merge_interleaved() {
+    new_ucmd()
+        .arg("-m")
+        .arg("merge_ints_interleaved_1.txt")
+        .arg("merge_ints_interleaved_2.txt")
+        .arg("merge_ints_interleaved_3.txt")
+        .succeeds().stdout_is_fixture("merge_ints_interleaved.expected");
```

Comment:
use stdout_only_fixture (which also asserts that stderr is empty) unless you don't know yet if there should be stderr output or not